website/blog/2023-12-04-0.73-debugging-improvements-stable-symlinks.md

```diff
@@ -0,0 +1,178 @@
+
+In React Native 0.73 we've addressed this issue. Hermes now captures all `console.log()` calls in the background, which will be sent to the Console tab when a debugger is first connected — matching the debugging experience in web browsers.
+
+#### Updated Debugging Docs
+
+We've refreshed the [Debugging](/docs/debugging) section of our docs, which now includes up-to-date information on how to connect all supported debuggers, more info on React DevTools, and refreshed visuals.
+
+![Overview of the Debugging docs as of 0.73](/blog/assets/0.73-debugging-docs.jpg)
+
+#### Experimental New Debugger
+
+The React Native team is working on a new JavaScript debugger experience, intended to replace Flipper, with a Technical Preview available as of React Native 0.73. The new debugger opens immediately, and features a stripped-back Chrome DevTools UI customized for debugging React Native with Hermes.
+
+:::note
+The new debugger is **experimental** and has some [known issues](https://github.com/react-native-community/discussions-and-proposals/discussions/733) we are actively working to solve in a future release of React Native.
+:::
+
+![The new debugger frontend opened to the "Welcome" pane](/docs/assets/debugging-debugger-welcome.jpg)
+
+Learn more about enabling this experience [in the docs](/docs/next/debugging?js-debugger=new-debugger#opening-the-debugger).
+
+### Stable Symlink Support in Metro
+
+Support for resolving symlinks in Metro is now enabled by default. Symlink support enables React Native to work with monorepo setups when containing directories are configured with [`watchFolders`](https://metrobundler.dev/docs/configuration/#watchfolders).
+
+Symlinks are deeply represented in Metro's internals, meaning they work with features such as [Fast Refresh](/docs/fast-refresh), and incur little performance overhead during bundling. Symlinks are supported on all desktop platforms, with and without Watchman.
+
+:::info
+#### Monorepo workarounds
+
+We are aware there are still edge cases when using React Native in a monorepo layout. We have planned work to address some of these, which didn't quite make it into 0.73 — but which we aim to ship as soon as possible.
+
+**For React Native template projects (`npx react-native init`)**, you will need to configure any [`watchFolders`](https://metrobundler.dev/docs/configuration/#watchfolders) outside of the project root in order for Metro to discover them ([more info](https://metrobundler.dev/docs/configuration/#unstable_enablesymlinks-experimental)). You may also need to update file paths if your `react-native` dependency is installed to a folder at a different level.
+
+**For Expo apps**, support for Yarn (Classic) workspaces is configured out of the box. See also the [Work with monorepos](https://docs.expo.dev/guides/monorepos/) guide in the Expo docs.
+:::
```

Comment:
IMHO this should be moved to a separate page or article that we should link here

Reply:
I agree it is an awkward block of text, but at the same time having this information inline is:

- Honest and up front about remaining limitations.
- Re-commits to shipping the remaining work soon (which is both looking very tangible (cc @robhogan), and follows RN EU talk guidance).
- Justified to include given we have named Stable Symlink Support as a headlining feature.
    - If we want to reconsider this, I'm open to it!